### PR TITLE
rotate the default ssh key

### PR DESCRIPTION
**Pull request by dmfine**
new key will be in passpack, passphrase held by dfine and a handful of others

### CODE REVIEW

File: h/env-prod.yml [hunk 1]
```diff
@@ -39,10 +39,10 @@ OptionSettings:
     SecurityGroups: sg-fdf9c19a,sg-a6073fc1
     IamInstanceProfile: aws-elasticbeanstalk-ec2-role
     InstanceType: m4.large
-    EC2KeyName: ops
+    EC2KeyName: ops-20191105
   aws:autoscaling:asg:
     MaxSize: '12'
-    MinSize: '1'
+    MinSize: '4'
```
Comment (robertknight) [inline h/env-prod.yml]: Hi @dmfine - Can you revert this change as it isn't related to the SSH key change. If you still want to propose it for other reasons, can you make that part of a separate PR.

Comment (dmfine) [reply h/env-prod.yml]: Fixed

**Comment by dmfine** [comment]
The key is in th 1password dev ops vault
